notificationWorker/db/db_test.go

```diff
@@ -0,0 +1,27 @@
+package db
+
+import (
+	"context"
+	"testing"
+
+	"github.com/slovak-egov/einvoice/notificationWorker/config"
+	"github.com/slovak-egov/einvoice/pkg/entity"
+)
+
+var ctx = context.Background()
+
+var conf = config.New()
+
+var connector = NewConnector(conf.Db)
```

Comment:
can we initialize config and connector in `main_test` similar to https://github.com/slovak-egov/einvoice/blob/main/apiserver/app/main_test.go?

We can also drop DB connection afterwards